clients/tfchain-client-go/contract_test.go

```diff
@@ -99,6 +99,53 @@ func TestGetRentContract(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func TestGetContractPaymentState(t *testing.T) {
+	cl := startLocalConnection(t)
+	defer cl.Close()
+
+	t.Run("non-existent contract", func(t *testing.T) {
+		_, err := cl.GetContractPaymentState(0)
+		require.Error(t, err)
+	})
+
+	t.Run("existing contract", func(t *testing.T) {
+		_, err := cl.GetContractPaymentState(1)
```

Comment:
I noticed that the tests are using hardcoded contract ID 1 for testing existing contracts, which isn't guaranteed to exist. Let's improve the tests by creating a contract first, then testing the functions with the actual contract ID.
Please make sure each test is independent and doesn't rely on external state.
With the current approach tests won't pass on CI